flixel/input/mouse/FlxMouse.hx

```diff
@@ -397,8 +397,8 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
 		//actually position the flixel mouse cursor graphic
 		if (visible)
 		{
-			cursorContainer.x = FlxG.game.mouseX;
-			cursorContainer.y = FlxG.game.mouseY;
+			cursorContainer.x = this.screenX;
```

Comment:
Just apply another commit and push it to the same branch and it will
automatically update the pull request.

On Thu, Apr 21, 2016 at 3:33 PM, Seanw265 <notifications@github.com> wrote:

> In flixel/input/mouse/FlxMouse.hx
> <https://github.com/HaxeFlixel/flixel/pull/1823#discussion_r60650477>:
>
> > @@ -397,8 +397,8 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
> >  		//actually position the flixel mouse cursor graphic
> >  		if (visible)
> >  		{
> > -			cursorContainer.x = FlxG.game.mouseX;
> > -			cursorContainer.y = FlxG.game.mouseY;
> > +			cursorContainer.x = this.screenX;
>
> After doing some testing, it seems
>
> cursorContainer.x = FlxG.game.mouseX * FlxG.scaleMode.mouseMultiplier.x;
> cursorContainer.y = FlxG.game.mouseY * FlxG.scaleMode.mouseMultiplier.y;
>
> works, while what is currently in the pull request does not. Is there a
> way for me to change that easily?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/HaxeFlixel/flixel/pull/1823/files/2856c3d11bb17603dd071e6de7cf66548a16f298#r60650477>
>



-- 
www.fortressofdoors.com -- Games, Art, Design